neo/Prompt/Commands/Wallet.py

```diff
@@ -109,12 +120,66 @@ def command_desc(self):
         return CommandDesc('create', 'creates a new NEO wallet address', [p1])
 
 
-class CommandWalletVerbose(CommandBase):
+class CommandWalletOpen(CommandBase):
 
     def __init__(self):
         super().__init__()
 
     def execute(self, arguments):
+        if PromptData.Wallet:
+            CommandWalletClose().execute()
+
+        path = get_arg(arguments, 0)
+
+        if path:
+
+            if not os.path.exists(path):
+                print("Wallet file not found")
+                return
+
+            passwd = prompt("[password]> ", is_password=True)
+            password_key = to_aes_key(passwd)
+
+            try:
+                PromptData.Wallet = UserWallet.Open(path, password_key)
+
+                PromptData.Prompt.start_wallet_loop()
+                print("Opened wallet at %s" % path)
+                return PromptData.Wallet
+            except Exception as e:
+                print("Could not open wallet: %s" % e)
+
+        else:
+            print("Please specify a path")
+
+    def command_desc(self):
+        p1 = ParameterDesc('path', 'path to open the wallet file')
+        return CommandDesc('open', 'opens a NEO wallet', [p1])
+
+
+class CommandWalletClose(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self):
+        if PromptData.Wallet:
```

Comment:
Maybe the implementation could be moved to a static method or to PromptData.
It could avoid having to instanciate CommandWalletClose to call execute. (`CommandWalletClose().execute()` in prompt.py).

What do you think?

Reply:
This is why I originally had it as a class method.  But changed it after your review. Couldn't we just keep it as a class method?

Reply:
A static method still required  me to instantiate the class

Reply:
I realize that my message was probably confusing, sorry for that.

I was suggesting to keep execute as is (to override `execute(self arguments)`) and have it call another static method.
Imo a `@classmethod` should only be used when you want to override it in a child class or when you need to have access to `cls`.

I think the following should work:

```python
class CommandWalletClose(CommandBase):
...
def execute(self, arguments=None):
    CommandWalletClose.close_wallet()


@staticmethod
def close_wallet():
    if not PromptData.Wallet:
        return False

    path = PromptData.Wallet._path
    PromptData.Prompt.stop_wallet_loop()
    PromptData.Wallet.Close()
    PromptData.Wallet = None
    print("Closed wallet %s" % path)
    return True
```

It should be possible to call it with `CommandWalletClose.close_wallet()`.
I think it would be even better to have this function in `PromptData` and call `PromptData.close_wallet()`.

Reply:
@LysanderGG Thank you. I will give it a shot. I prefer this way because it makes sense to me where it is located. I wouldn't intuitively think to look for the function called by `wallet close` in `PromptData`. But, that is my opinion.